projects/aca-content/src/lib/components/bulk-actions-dropdown/bulk-actions-dropdown.component.scss

```diff
@@ -0,0 +1,31 @@
+/* stylelint-disable selector-class-pattern */
+.aca-bulk-select {
+  margin-top: 31px;
+}
+
+.aca-bulk-form-field {
+  margin-left: 24px;
+  margin-top: 1.4375em;
+  width: 295px;
+
+  .mat-form-field-flex,
```

Comment:
As part of the angular migration, we also added a mat-selectors file, that contains all angular material selectors. The plan is to use the references from that file, instead of directly referring to .mat selectors in our css (I assume that is what you meant when you said 'will be removed after Angular migration). 

However, I am wondering how we will keep track of all the material selectors that were added/are being added after the angular migration was started (such as this one). Maybe we can add TODO statements, and then do a project wide search after the ng migration merge. Or we can keep references in a sepearate single file. 
@MichalKinas @DenysVuika what do you think?

Reply:
Sounds good, fortunately ADF should be merged today and ACA will be next so this PR will be merged after migration one

Reply:
added TODO and created story for it that will be done in the end of Legal Hold development ACS-8458